prov/gni/src/gnix_dom.c

```diff
@@ -427,8 +433,8 @@ gnix_domain_ops_open(struct fid *fid, const char *ops_name, uint64_t flags,
 	return ret;
 }
 
-int gnix_domain_open(struct fid_fabric *fabric, struct fi_info *info,
-		     struct fid_domain **dom, void *context)
+DIRECT_FN int gnix_domain_open(struct fid_fabric *fabric, struct fi_info *info,
```

Comment:
Okay --  I know why some of these don't have STATIC.  They are assigned to the function table in fabric.c.  Please ignore my above comments.